src/data-pipeline/spark-etl/gradle.properties

```diff
@@ -1,17 +1,17 @@
 # Project
 group=software.aws.solution
-projectVersion=1.1.0
+projectVersion=1.1.1
 
 # Dependencies
-lombokVersion=1.18.26
-junitVersion=5.9.2
-junitBomVersion=5.9.2
+lombokVersion=1.18.30
+junitVersion=5.10.1
+junitBomVersion=5.10.1
 assertJVersion=3.24.2
-mockitoVersion=5.2.0
-jacocoVersion=0.8.8
+mockitoVersion=5.8.0
+jacocoVersion=0.8.11
 jacocoLogVersion=3.1.0
-checkstyleVersion=9.2.1
-guavaVersion=31.1-jre
-sparkVersion=3.3.2
+checkstyleVersion=9.3
+guavaVersion=32.1.2-jre
+sparkVersion=3.4.1
 uapJavaVersion=1.5.3
```

Comment:
Why not bump it to `1.6.1`?

Reply:
1.6.1 and even 1.5.4 do not work, spark job will fail because of snakeyaml. 

new version uap has dependency of org.yaml.snakeyaml-2.x, but the EMR has a dependency of org.yaml.snakeyaml-1.xx.